requirements-base-runtime.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.12
```

Comment:
I don't think this should be the case just yet 😄

Reply:
I upgraded to Ubuntu 24.04 yesterday, and yes Python 3.12 makes more than just a cosmetic impact.

Reverted with 27dde674a :+1:

Reply:
Pro-tip: Use `pyenv`. We do have `.python-version` files in our repos, which controls `pyenv` to automatically switch to the right version. :)